docs/contracts.rst

```diff
@@ -432,13 +432,18 @@ value types and strings.
         bytes32 constant myHash = keccak256("abc");
     }
 
+.. index:: ! functions
 
-.. _view-functions:
+.. _functions:
 
-**************
-View Functions
-**************
+*********
+Functions
+*********
+
+.. index:: ! _view-functions
```

Comment:
I think with this the `view-functions` link is gone?

Reply:
Yes, sorry will reinstate.